torchtext/datasets/text_classification.py

```diff
@@ -155,6 +158,10 @@ def AG_NEWS(*args, **kwargs):
     Separately returns the training and test dataset
 
     Arguments:
+        tokenizer: the tokenizer used to preprocess raw text data.
```

Comment:
Is this part of IMDB? Could you clarify the scope of this PR?

Reply:
We change the API to keep those text classification datasets consistent.

Reply:
Since we add `tokenizer` to `_setup_datasets`, it affects all the other datasets here.